### PR TITLE
docs: add instructions for CSS var. autocomplete

### PR DESCRIPTION
**Pull request by booc0mtaco**
### Summary:

Add a small block of text to describe how to enable this nice DX improvement.

### Test Plan:

- n/a (documentation-only change)

### CODE REVIEW

**Review by jeremiah-clothier — COMMENTED** [review]
Do we need this new file? The variables exist in `@chanzuckerberg/eds/index.css`

**Comment by booc0mtaco** [comment]
> Do we need this new file? The variables exist in `@chanzuckerberg/eds/index.css`

I tried using that, and some error popup appeared. Testing it now, it seems to work? Maybe I typed the path incorrectly or something weird happened when testing...

Since using index works, that's a better choice. keeps the bundle size down and the instructions simpler. I'll update this to point to `index.css` and remove the new file generation.

File: README.md [hunk 1]
```diff
@@ -51,6 +51,23 @@ Refer to the [tokens tailwind section][tokens] for usage guidelines.
...
 [tokens]: https://chanzuckerberg.github.io/edu-design-system/?path=/docs/documentation-guidelines-tokens--docs
 
+### CSS Variable Setup
+
+EDS also provides the tokens used in the internal styles, to use in any custom component recipes and designs. If using VSCode, you can set up the IDE to expose the token values and perform autocomplete:
+
+1. Install the [CSS Var Complete](https://marketplace.visualstudio.com/items?itemName=phoenisx.cssvar) VSCode extension
+2. Add the following setting to your user or workspace settings file
+
+```json
```
Comment (jeremiah-clothier) [inline README.md:61]:
```suggestion
```jsonc
```

This may get rid of the red highlighting in the `.md` file

**Review by jeremiah-clothier — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (booc0mtaco) [reply README.md]: Cool hadn't used `jsonc` before. Thanks !

**Pull request merged**